### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by AminaBuhari**
In this project, I populated the tables and also queried multiple tables.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+
+CREATE TABLE owners (
+    id            SERIAL PRIMARY KEY,
+    full_name TEXT NOT NULL;
+    age INT NOT NULL; 
+)
+
+CREATE TABLE species (
+    id        SERIAL PRIMARY KEY,
+   name TEXT NOT NULL;
```
Comment (andersonlebon) [inline schema.sql]: image

- [ ] kindly user comas `,` at the end of the line in the `()`, note that `; ` are user at the end of the full instruction(query). Hint example :point_down: 

```sql
CREATE TABLE owners (
    id            SERIAL PRIMARY KEY,
    full_name TEXT NOT NULL;
    age INT NOT NULL; 
)
```
this one :point_down:  is better and does not throw errors 

```sql

CREATE TABLE owners (
    id            SERIAL PRIMARY KEY,
    full_name TEXT NOT NULL,
    age INT NOT NULL,
);
```

File: schema.sql [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+-- Modify animals table:
+
+--     Make sure that id is set as autoincremented PRIMARY KEY
+
+
+
+BEGIN;
+ALTER TABLE Animals DROP CONSTRAINT "GENERATED BY DEFAULT AS IDENTITY" FROM id;
+ALTER TABLE Animals ADD CONSTRAINT "SERIAL" PRIMARY KEY (id);
+COMMIT;
```
Comment (andersonlebon) [inline schema.sql]: image

- [ ] Kindly this code is throwing an error kindly correct it.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @AminaBuhari 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (AminaBuhari) [reply schema.sql]: Thanks for pointing out the syntax error. Updated!

File: schema.sql (hunk 2 above)
Comment (AminaBuhari) [reply schema.sql]: Thank you for pointing this out. I've removed this line of code. The Constraint "GENERATED BY DEFAULT AS IDENTITY" earlier added while creating the column ID has an implicit sequence that will add and set a value serially if not given.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @AminaBuhari , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well implemented required changes.

### Optional suggestions :arrow_down_small: 

- [ ] The commit message should be written in the imperative tense and the first letter should be capitalized also.. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] Pull Request should have a descriptive title telling you what changes are inside. Please update your PR title.

- [ ] Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**